Style Guide and Glossary cross-reference edits (A-C) (#5762)
braze-inc/braze-docs: _docs/_hidden/other/dp-technical-assistance.md

# Data Protection Technical Assistance

> There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the  as amended by the California Privacy Rights Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.

Reviewer: CCPA or CPRA?

Author: I think it's meant to be "the California Consumer Privacy Act ("**CCPA**") as amended by the California Privacy Rights Act (“**CCPA**”)"

However, I just saw the note "@@ -13,7 +13,7 @@ Warning! Don't make any changes to this document without approval from the legal" so maybe this shouldn't be edited at all?

> There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the  as amended by the California Privacy Rights Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.

Reviewer: ```suggestion
There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act  (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```
```suggestion
There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```

> There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the  as amended by the California Privacy Rights Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.

Reviewer: ```suggestion
There are a range of data protection laws that regulate what organizations can do with personal data (“**Data Protection Laws**”), including the EU and UK General Data Protection Regulation (“**GDPR**”), the California Consumer Privacy Act (“**CCPA**”) and the Health Insurance Portability and Accountability Act (“**HIPAA**”). There are other national, state, and industry-specific Data Protection Laws and regulations that may apply to your business.
```
